diff --git a/samurai/madds.f90.in b/samurai/madds.f90.in
index efdd722..2207207 100644
--- a/samurai/madds.f90.in
+++ b/samurai/madds.f90.in
@@ -1,7 +1,7 @@
 module madds
    use precision, only: ki, ki_ql, ki_lt
 @case_with_golem@use precision_golem, only: ki_gol => ki
-@case_with_avh@use avh_olo_kinds, only: ki_avh => kindr2
+@case_with_avh@use avh_olo_dp_kinds, only: ki_avh => kindr2
    use constants
    use options
    use mfunctions